scripts/build_tree.py

```
@@ -479,6 +478,182 @@ def _build_flutter_windows() -> int:
+                 dst_paths: list[Path], extra_defines: list[str] | None = None) -> int:
+    """CMake 配置 + 构建 (x64 Release) + 复制 DLL。"""
+    build_dir = src / "build" / "x64"
+    build_dir.mkdir(parents=True, exist_ok=True)
+    defines = ["-DCMAKE_BUILD_TYPE=Release"] + (extra_defines or [])
+
+    cmake = _find_vs_cmake() or shutil.which("cmake") or "cmake"
+
+    # VS Generator 名 (用 vswhere 动态获取版本号)
+    vs_gen = _get_vs_generator()
+    if vs_gen:
+        cmake_args = ["-G", vs_gen, "-A", "x64"] + defines
```

Ruff (0.15.15)

[warning] Consider `["-G", vs_gen, "-A", "x64", *defines]` instead of concatenation

Replace with `["-G", vs_gen, "-A", "x64", *defines]`

(RUF005)